test/utils/runners.go

```diff
@@ -851,6 +851,34 @@ waitLoop:
 	return nil
 }
 
+// Wait up to 10 minutes for at least 'replicas' many pods to be Running and at least
+// one matching pod exists.
+func WaitForEnoughPodsWithLabelRunning(c clientset.Interface, ns string, label labels.Selector, replicas int) error {
```

Comment:
Please merge this function with the already existing (e.g. by passing function that checks if the number of running pods is correct or sth like that).

Reply:
Good idea (I was too lazy to do it earlier:)
LGTY?